app/src/main/java/com/example/hackernews/view/activities/MainActivity.kt

```diff
@@ -138,16 +182,19 @@ class MainActivity :
                 binding.drawerLayout.closeDrawer(GravityCompat.START)
             }
             R.id.side_feedback -> {
-                Toast.makeText(this, "Clicked on feedback", Toast.LENGTH_LONG).show()
+                writeToast("Clicked on feedback")
             }
             R.id.side_saved_stories -> {
                 binding.drawerLayout.closeDrawer(GravityCompat.START)
+                viewModel.loadSavedStories() // this method works, but there is one problem. It works only when first task(loading api news) is finished//
```

Comment:
You can have a property in ViewModel that holds the current type of news selected by the user. This way you know which news you should show.